account_analytic_distribution_manual/views/account_analytic_distribution_manual_views.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="view_account_analytic_distribution_manual_tree" model="ir.ui.view">
+        <field name="name">view.account.analytic.distribution.manual.tree</field>
+        <field name="model">account.analytic.distribution.manual</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+                <field
+                    name="analytic_distribution"
+                    widget="analytic_distribution"
+                    optional="show"
+                />
+                <field name="company_id" groups="base.group_multi_company" />
+            </tree>
+        </field>
+    </record>
+
+    <record id="view_account_analytic_distribution_manual_form" model="ir.ui.view">
+        <field name="name">view.account.analytic.distribution.manual.form</field>
+        <field name="model">account.analytic.distribution.manual</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <widget
+                        name="web_ribbon"
+                        title="Archived"
+                        bg_color="bg-danger"
+                        attrs="{'invisible': [('active', '=', True)]}"
+                    />
+                    <div class="oe_title">
+                        <label for="name" />
+                        <h1>
+                            <field name="name" placeholder="Name..." />
+                        </h1>
+                    </div>
+                    <group colspan="2">
+                        <field name="active" invisible="1" />
+                        <field name="company_id" groups="base.group_multi_company" />
+                        <field
+                            name="analytic_distribution"
```

Comment:
I thinks it should be required, as it not make any sense to create a configuration without analytic distribution